test/Table_Tests/src/Database/SQLite_Spec.enso

```diff
@@ -172,4 +172,11 @@ spec =
 
         file.delete_if_exists
 
+        Test.specify 'should not duplicate warnings' <|
```

Comment:
This feels more like integration test but at least it demonstrates that we don't generate rubbish anymore.

Reply:
I think it's good to have such integration tests.